servlet/src/test/java/io/undertow/servlet/test/security/form/FormAuthenticationRootContextRedirectTestCase.java

```diff
@@ -0,0 +1,123 @@
+/**
```

Comment:
I think we are missing the first block in the license header? Is there a reason for this or was it mistakenly removed?

Reply:
Looks like I had wrong template in the IDE, I'll change that